loadtest/control/simulcontroller/actions.go

```diff
@@ -825,13 +825,18 @@ func (c *SimulController) addReaction(u user.User) control.UserActionResponse {
 		}
 	}
 
-	if reactionLimit := u.Store().Config().ServiceSettings.UniqueEmojiReactionLimitPerPost; reactionLimit != nil {
```

Comment:
I think there's a similar line in `gencontroller`. Wondering if it's worth fixing that to avoid hitting it again.

Reply:
Done